src/steps/click-on-element.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, RunStepResponse, FieldDefinition, StepDefinition } from '../proto/cog_pb';
+
+export class ClickOnElement extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Click an element on a page';
+  // tslint:disable-next-line:max-line-length
+  protected stepExpression: string = 'click on page element (?<domQuerySelector>.+)';
```

Comment:
@amcpanaligan Reading this again, I think `click on` sounds a little old-fashioned.  Perhaps we just use `click the page element`